ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -901,6 +998,38 @@ private void assertClosed() {
     Assert.assertEquals(e.getErrorCode(), RouterErrorCode.RouterClosed);
   }
 
+  /**
+   * Does the TTL update test by putting a blob, checking its TTL, updating TTL and then rechecking the TTL again.
+   * @param numChunks the number of chunks required when the blob is put. Has to divide {@link #PUT_CONTENT_SIZE}
+   *                  perfectly for test to work.
+   * @throws Exception
+   */
+  private void doTtlUpdateTest(int numChunks) throws Exception {
```

Comment:
`TtlUpdateManagerTest` has a test for when servers return the `Deleted` error code. I think that should be enough - anything more that adding this could add?